dockstore-common/src/test/resources/fixtures/dockstore12.yml

```diff
@@ -90,3 +90,4 @@ notebooks:
       - /other0
   - name: notebook1
     path: /notebook1.ipynb
+    image: quay.io/seqware/seqware_full/1.1
```

Comment:
Should this be a non-Docker example to emphasize how this may be different from what is normally stored in `Image`?